dotnet-datetime/StopWatchCSharp/StopWatchCSharp/StopWatchMethods.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Diagnostics;
+
+namespace StopWatchCSharp
+{
+    public class StopWatchMethods
+    {
+        public Stopwatch BubbleSort(int[] numArray, int size)
+        {
+            var stopWatch = new Stopwatch();
+            
+            stopWatch.Start();
+
+            for (int i = 0; i < size - 1; i++)
+                for (int j = 0; j < size - i - 1; j++)
+                    if (numArray[j] > numArray[j + 1])
+                    {
+                        var tempVar = numArray[j];
+                        numArray[j] = numArray[j + 1];
+                        numArray[j + 1] = tempVar;
+                    }
+
+            stopWatch.Stop();
+
+            return stopWatch;
+        }
+
+        public Stopwatch HeapSort(int[] array, int size)
+        {
+            var stopWatch = new Stopwatch();
+
+            stopWatch.Start();
+
+            if (size <= 1) 
+            {
+                stopWatch.Stop();
+
+                return stopWatch;
+            }
+                
+            for (int i = size / 2 - 1; i >= 0; i--)
+            {
+                Heapify(array, size, i);
+            }
+
+            for (int i = size - 1; i >= 0; i--)
+            {
+                var tempVar = array[0];
+                array[0] = array[i];
+                array[i] = tempVar;
+
+                Heapify(array, i, 0);
+            }
+
+            stopWatch.Stop();
+
+            return stopWatch;
+        }
```

Comment:
We should create a master method that initializes, starts and stops the `StopWatch` and pass in the method that we want to time. It would probably be best to create a new class called `SortingMethods` that contains these two sorts. Then this class can be called something like `ExecutionTimeMeasurer` or something like that. It would have a single method that takes a sort `Action` and the arguments to call the sort methods. Then internally it would create, start and stop the `StopWatch` timing the execution of the sort. It could be even more generic and we could call it `BenchmarkTimer` or something, which would take just an `Action` to execute. Then the caller could pass in whatever they want as a lambda or whatever.